gzipi/lib.py

```diff
@@ -383,7 +531,9 @@ def _start_of_line(fin, lineterminator=b'\n', bufsize=io.DEFAULT_BUFFER_SIZE):
             break
 
 
-def _buffer_chunk(fin, start, end, pivot, lineterminator):
+def _buffer_chunk(
+    fin: IO[bytes], start: int, end: int, pivot: int, lineterminator: bytes
```

Comment:
A bit hard to read the function signature.

```suggestion
    fin: IO[bytes],
    start: int,
    end: int,
    pivot: int,
    lineterminator: bytes,
```